sed/calibrator/energy.py

```diff
@@ -108,7 +110,9 @@ def __init__(
         ) / 2 ** (self.binning - 1)
         self.tof_fermi = self._config["energy"]["tof_fermi"] / 2 ** (self.binning - 1)
         self.color_clip = self._config["energy"]["color_clip"]
-
+        self.sector_delays = self._config["dataframe"].get("sector_delays", None)
+        self.sector_id_column = self._config["dataframe"].get("sector_id_column", None)
```

Comment:
there is a check later. I adapted these to follow the structure you introduced, but I find this rather convoluted and easier to make mistakes. I would rather keep parameter searching in the functions that need them, instead of thousand lines away.

Reply:
there is a check later.